packages/number-field/src/vaadin-number-field-mixin.js

```diff
@@ -204,10 +206,21 @@ export const NumberFieldMixin = (superClass) =>
       // it means scrolling is in progress, therefore we shouldn't update field value.
       if (e.cancelable) {
         e.preventDefault();
+        this.__blurActiveElement();
         this._increaseValue();
       }
     }
 
+    /** @private */
+    __blurActiveElement() {
+      // If another element is focused, blur it on step button touch
```

Comment:
```suggestion
      // If another element is focused, blur it on step button touch to hide mobile keyboard that might still be open for the other element
```
Could provide some more context here already, though the issue link also helps.

Reply:
Thanks, makes sense. Updated the comment.